test/runtests.jl

```diff
@@ -1,5 +1,30 @@
 using Test
 
-@testset "Stand-in test" begin
-    @test 1 == 1
-end
+using ClimaLSM
+using ClimaLSM.Domains: Column
+include(joinpath("../src", "Soil.jl"))
```

Comment:
Can do `using ClimaLSM.Soil`, unless we'd rather not include the subcomponent models in the `ClimaLSM` module in anticipation of splitting the repo later.

Reply:
im good with whatever you think makes it easier to develop now!